site/content/docs/5.0/examples/features/index.html

```diff
@@ -7,7 +7,7 @@
 ---
 
 <svg xmlns="http://www.w3.org/2000/svg">
-  <symbol id="bootstrap" viewBox="0 0 118 94">
+  <symbol id="bootstrap" viewBox="0 0 118 94" style="display:none">
```

Comment:
Why only hide it? Couldn't you replace SVG content with Orange's logo?
Didn't check at all where this is used, BTW :')